### ISSUE TITLE
Misleading example of AST for TokenTypes.CLASS_DEF

### ISSUE THREAD

**Issue by 20001LastOrder** [checkstyle/checkstyle#9379]
Hi, I'm working on adding an example of AST for TokenTypes.ABSTRACT. But I noticed that there may be an issue with the example for TokenTypes.CLASS_DEF.

The example is like the following:
![image](https://user-images.githubusercontent.com/25534730/110816228-276bab80-8259-11eb-866a-71b003baa89d.png)

But I am not sure where does the `+--EXTENDS_CLAUSE` comes from...

---

When I try the example in the terminal, it gives me t he following:

```bash
>java -jar checkstyle-8.41-all.jar -t MyClass.java
CLASS_DEF -> CLASS_DEF [1:0]
|--MODIFIERS -> MODIFIERS [1:0]
|   `--LITERAL_PUBLIC -> public [1:0]
|--LITERAL_CLASS -> class [1:7]
|--IDENT -> MyClass [1:13]
|--IMPLEMENTS_CLAUSE -> implements [2:0]
|   `--IDENT -> Serializable [2:11]
`--OBJBLOCK -> OBJBLOCK [3:0]
    |--LCURLY -> { [3:0]
    `--RCURLY -> } [4:0]
```
MyClass.java file:
``` bash
>cat MyClass.java
public class MyClass
implements Serializable
{
}
```
---
If this is an issue, then I can quickly fix it. Thanks

**Comment by nmancus1** [comment]
This is a good find. Please update issue description to add `cat MyClass.java` to show the exact file you are parsing.

**Comment by 20001LastOrder** [comment]
I've added the content in MyClass.java in the description. Thank you.

**Comment by rnveach** [comment]
Looks like `EXTENDS` is conditional if it is written or not in the source.

````
CLASS_DEF -> CLASS_DEF [1:0]
|--MODIFIERS -> MODIFIERS [1:0]
|  `--LITERAL_PUBLIC -> public [1:0]
|--LITERAL_CLASS -> class [1:7]
|--IDENT -> MyClass [1:13]
|--EXTENDS_CLAUSE -> extends [1:21]
|  `--IDENT -> Object [1:29]
````

**Comment by AbhinavRajputEXE** [comment]
can I work on this?

**Comment by nmancus1** [comment]
> can I work on this?

Sure, please send PR.

**Comment by AbhinavRajputEXE** [comment]
Hey, @nmancus1, after reading the issue desc. , It looks like the problem was that, the AST was in the Old format and with that extends keyword.
![Screenshot (598)](https://user-images.githubusercontent.com/80591258/144596877-7350ad99-eb22-4333-8e0e-e14987290dc0.png)
But it looks like this was already fixed, so what exactly do I need to change in this now.

**Comment by nmancus1** [comment]
@AbhinavRajputEXE thanks for investigating.

> ... it looks like this was already fixed

You are correct, I can see that this issue was closed via #9408
